_shared/project/tox.ini

```diff
@@ -133,9 +137,9 @@ commands =
     lint: pydocstyle src tests bin
     lint: pycodestyle src tests bin
 {% endif %}
-    tests: coverage run -m pytest --failed-first --new-first --no-header --quiet {posargs:tests/unit/}
+    tests: python -m pytest --cov --cov-report= --cov-fail-under=0 {%- if cookiecutter.get("__parallel_unit_tests") %} --numprocesses logical {%- endif %} --failed-first --new-first --no-header --quiet {posargs:tests/unit/}
```

Comment:
Updated unit tests command now that we're using `pytest-cov` and have optional support for running the unit tests in parallel.

* `--cov`: needed to enable `pytest-coverage`, even with the other `--cov-*` options being here
* `--cov-report=` (with no argument) disables coverage reporting. We don't want `make test` (or `tox -e tests` etc) to print out a coverage report. Especially not when you only run certain tests (`tox -e tests tests/unit/foo_test.py`), it's really annoying. Instead, `make coverage`, `make sure`, or `tox -e coverage` will print the coverage report
* `--cov-fail-under`: we also don't want `make test` (or `tox -e tests` etc) to fail if coverage is not 100%, especially not when you only run certain tests (`tox -e tests tests/unit/foo_test.py`). Instead, `make coverage`, `make sure`, or `tox -e coverage` will still fail if coverage is not 100%.